ckanext/gla/templates/organization/read.html

```diff
@@ -51,6 +51,9 @@
         <div class="page_primary_action">
             <div class="row justify-content-between">
                 <div class="col-md-3">
+                    {{ h.snippet ('snippets/add_dataset.html', group=group_dic) }}
```

Comment:
Should this also have the check that you've got in `search.html`?
This bit: `{% if h.check_access('package_create') %}`

(It looks like the [read template in the base CKAN repo](https://github.com/ckan/ckan/blob/master/ckan/templates/organization/read.html#L5) has also got the `check_access` call)